misc-utils/Makemodule.am

```diff
@@ -211,3 +211,15 @@ fincore_SOURCES = misc-utils/fincore.c
 fincore_LDADD = $(LDADD) libsmartcols.la libcommon.la
 fincore_CFLAGS = $(AM_CFLAGS) -I$(ul_libsmartcols_incdir)
 endif
+
+if BUILD_HARDLINK
+usrbin_exec_PROGRAMS += hardlink
+hardlink_SOURCES = misc-utils/hardlink.c
+hardlink_LDADD = $(LDADD) libcommon.la
+hardlink_CFLAGS = $(AM_CFLAGS)
+#if HAVE_TINFO
```

Comment:
It seems  HAVE_TINFO is copy & past bug, right? I guess HAVE_PCRE here :-)

Reply:
Oops, I pushed an update.